components/raftstore/src/store/snap.rs

```diff
@@ -2013,7 +2013,7 @@ impl TabletSnapManager {
                 format!("{} should be a directory", path.display()),
             ));
         }
-        file_system::clean_up_trash(&path)?;
+        file_system::clean_up_exclude_prefix(&path, SNAP_REV_PREFIX)?;
```

Comment:
It's safer to use allowlist instead of blocklist.

Reply:
fixed